### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by Kelvin-Ben**
### Hello Microverse Project
- In this project, my main goal was to set up a new repository and create branches.
- I added the Html/CSS linters and other few files
- I created an index.html file and linked a style.css file to it so that effects could be seen.

### CODE REVIEW

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Kevo-hacker 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+👤 **Kelvin Ben**
+
+- GitHub: [@Kevo-hacker] (https://github.com/Kevo-hacker)
+- Twitter: [@kevoyout144] (https://twitter.com/kevoyout144)
+- LinkedIn: [Kelvin Ben] (https://www.linkedin.com/in/kelvin-ben-323043173/)
+
+👤
```
Comment (muneebulrehman) [inline README:18]: - Optional
Remove this icon.

**Review by muneebulrehman — COMMENTED** [review]
Hi @Kevo-hacker , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README [hunk 2]
```diff
@@ -0,0 +1,39 @@
+
+# Hello-Microverse
```
Comment (ha-manel) [inline README:2]: [OPTIONAL]
- [x] Please consider adding 'md' extension to the file (README.md) in order to apply markdown that you've written, it makes the README file look more readable and more professional

**Review by ha-manel — COMMENTED** [review]
## Hi @Kevo-hacker 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
* Well written PR description ✔️ 
* Proper use of GitHub flow ✔️  
* All tests passed ✔️ 

## Optional suggestions
* It's better and more professional if commit messages are in the imperative form (example: 'Update readme file').

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**